src/components/project/ProjectsImportDialog.vue

```diff
@@ -0,0 +1,317 @@
+<template>
+  <div class="ProjectsLoadDialog">
+    <v-dialog v-model="state.dialog" max-width="1024">
+      <v-card>
+        <v-card-title v-text="'Import projects'" />
+        <v-card-subtitle v-text="'Select a source'" />
+
+        <v-card-text>
+          <v-row class="mb-1">
+            <v-col cols="2">
+              <v-btn-toggle dark dense mandatory v-model="state.source">
+                <v-btn
+                  :key="item.value"
+                  :title="item.title"
+                  :value="item.value"
+                  v-for="item in state.items"
+                >
+                  <v-icon v-text="item.icon" />
+                </v-btn>
+              </v-btn-toggle>
+            </v-col>
+            <v-col class="pa-3" cols="10">
```

Comment:
```suggestion
            <v-col cols="auto">
              <v-btn-toggle dark dense mandatory v-model="state.source">
                <v-btn
                  :key="item.value"
                  :title="item.title"
                  :value="item.value"
                  v-for="item in state.items"
                >
                  <v-icon v-text="item.icon" />
                </v-btn>
              </v-btn-toggle>
            </v-col>
            <v-col class="pa-3" cols="9">
```
![grafik](https://user-images.githubusercontent.com/53972736/127016922-0427dd54-b5bc-4334-bfdd-184417c147e6.png)
or at least
```suggestion
            <v-col cols="3">
              <v-btn-toggle dark dense mandatory v-model="state.source">
                <v-btn
                  :key="item.value"
                  :title="item.title"
                  :value="item.value"
                  v-for="item in state.items"
                >
                  <v-icon v-text="item.icon" />
                </v-btn>
              </v-btn-toggle>
            </v-col>
            <v-col class="pa-3" cols="9">
```

![grafik](https://user-images.githubusercontent.com/53972736/127017209-780de231-a0b9-4f72-a628-2e0b80c2b28c.png)


seemed to work better. What does this look like when you use your setup?